bigchaindb/voter.py

```diff
@@ -190,3 +190,69 @@ def start(self):
         p_validate.start()
         p_vote.start()
         p_update.start()
+
+
+class Election(object):
+
+    def __init__(self, q_block_new_vote):
+        """
+        Initialize the class with the needed queues.
+
+        Initialize a queue where blocks with new votes will be held
+        """
+        self.q_block_new_vote = q_block_new_vote
+        self.q_blocks_with_quorum = mp.Queue()
+        self.q_invalid_blocks = mp.Queue()
+
+    def check_for_quorum(self):
+        """
+        Checks if block has enough invalid votes to make a decision
+        """
+        while True:
+            next_block = self.q_block_new_vote.get()
+            n_voters = len(next_block['block']['voters'])
+
+            vote_list = [vote['vote']['is_block_valid'] for vote in next_block['votes']]
+
+            n_invalid_votes = vote_list.count(False)
+
+            if n_invalid_votes > int(n_voters/2):
```

Comment:
We need to be careful with a split vote situation. A block is valid if the majority `n/2 + 1` says its valid and we may have a even number of nodes in the federation.

Image the case in which the federation has 10 nodes, 5 vote valid and 5 vote invalid. We have a split vote situation.
To be on the safe side in such a situation we should consider the block invalid, reassign the transactions and let the system try again.